extensions/notebook/src/test/book/book.test.ts

```diff
@@ -7,7 +7,7 @@ import * as vscode from 'vscode';
 import * as should from 'should';
 import * as TypeMoq from 'typemoq';
 import * as path from 'path';
-import * as fs from 'fs';
+import * as fs from 'fs-extra';
```

Comment:
>extra [](start = 24, length = 5)

Why move to this? Do the async methods get us anything useful vs just using the Sync methods on fs?

Reply:
@Charles-Gagnon  Wasn't sure if the test was timing out because of the Sync methods. I do plan on adding a couple more folders and files for future unit tests. Would you suggest using async or sync methods?

Reply:
I recommended this. Her method was already async, so using the async functions may help.

Reply:
I guess I don't really have an issue with this but you're awaiting the async calls anyways so I don't really see the benefits of using that over the specific sync methods. 

---
In reply to: [311735466](https://github.com/microsoft/azuredatastudio/pull/6633#discussion_r311735466) [](ancestors = 311735466)